### PR TITLE
add logup test &fix overflow issue

### PR DESCRIPTION
**Pull request by DreamWuGit**
refer to issue #143

### CODE REVIEW

File: circuit-std-rs/src/logup.rs [hunk 1]
```diff
@@ -486,12 +497,20 @@ pub fn query_count_by_key_hint<F: Field>(inputs: &[F], outputs: &mut [F]) -> Res
 pub fn rangeproof_hint<F: Field>(inputs: &[F], outputs: &mut [F]) -> Result<(), Error> {
     let n = inputs[0].to_u256().as_i64();
     let m = inputs[1].to_u256().as_i64();
-    let mut a = inputs[2].to_u256().as_i64();
+    //let mut a = inputs[2].to_u256().as_i64();
```
Comment (zhenfeizhang) [inline circuit-std-rs/src/logup.rs:500]: remove

**Review by zhenfeizhang — APPROVED** [review]
